runners/google-cloud-dataflow-java/pom.xml

```diff
@@ -122,6 +122,50 @@
         </plugins>
       </build>
     </profile>
+
+    <!-- A profile that adds an integration test phase for streaming if and only if
+         the streamingValidatesRunnerPipelineOptions maven property has been set.
+         It should be set to a valid PipelineOptions JSON string. -->
+    <profile>
```

Comment:
Having built this as described - a single Jenkins job that runs both - it takes many hours and should be parallelized. I believe @jasonkuster supports two separate postcommits until we have pipelined jobs, in which case they will still remain separate pieces of the pipeline.